.evergreen/config.yml

```diff
@@ -555,12 +555,14 @@ functions:
       params:
         working_dir: "src"
         script: |
-          # Disable xtrace (just in case it was accidentally set).
-          set +x
```

Comment:
Can this be added back now?

Reply:
I believe we want xtrace since we are running tox from here directly instead of from a separate bash file like before.

Reply:
This is here to avoid leaking info on the next line: `. ./prepare_atlas_connectivity.sh`

Reply:
Ah I see, makes sense!